### DISCUSSION TITLE
Emanote logo

### DISCUSSION THREAD

**Discussion by srid** [srid/emanote#379] [General]
The current logo is something I lazily took from svgrepo.com, IIRC. 

Ideally, I'd like to see the creation of a unique logo for Emanote. Does anybody wish to contribute their design fu?

**Comment by srid** [comment]
Went with https://www.svgrepo.com/svg/267765/paper-plane but open to ideas.

Why paper plane? Because a paper plane too *emanates* from paper. I also like the colourfulness.

**Reply by srid** [reply]
I am curious now. Is it possible to change the colour of the logo based on YAML config's `theme`?

**Reply by dezren39** [reply]
necro-comment: yeah definitely, guessing based on the other svg in the footer.tpl that you figured that part out too. these are giant because i just edited the source directly from inspect view sorry.
![paper-plane-green-hollow](https://github.com/user-attachments/assets/33bd9db1-1b29-46c7-b54f-afb1e33808b0)
![paper-plane-green-full](https://github.com/user-attachments/assets/6326c288-7fd4-4931-aad9-be874cbe0aea)
